css-grid-classes/style.css

```diff
@@ -0,0 +1,19 @@
+
+.container{
+
+  margin-left: auto;
+  margin-right: auto;
+  width: 980px;
+}
+.row{
+  display:flex;
+  flex-direction: row;
+  align-content: center;
+}
+img{
+  width: 100%;
+}
+.column-third{
+  width: calc(100%/3)
+
+}
```

Comment:
You're also going to need a `.column-fourth` and a `.column-two-thirds` classes. These will be used for the other sections with different widths.